samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -170,8 +181,8 @@ static void init_lc3(void)
 	freq_hz = bt_codec_cfg_get_freq(&codec_configuration.codec);
 	frame_duration_us = bt_codec_cfg_get_frame_duration_us(&codec_configuration.codec);
 	octets_per_frame = bt_codec_cfg_get_octets_per_frame(&codec_configuration.codec);
-	frames_per_sdu = bt_codec_cfg_get_frame_blocks_per_sdu(audio_stream.codec, true);
-	octets_per_frame = bt_codec_cfg_get_octets_per_frame(audio_stream.codec);
+	frames_per_sdu = bt_codec_cfg_get_frame_blocks_per_sdu(streams[0].codec, true);
+	octets_per_frame = bt_codec_cfg_get_octets_per_frame(streams[0].codec);
```

Comment:
Why those cannot be obtained from `&codec_configuration.codec` the same way as parameters above?

Reply:
They should be - Updated :)